parameter_files/fates_params_default.cdl

```diff
@@ -930,6 +931,13 @@ data:
   "sapwood",
   "structure" ;
 
+ fates_landuseclass_name =
+  "primaryland",
```

Comment:
I like having these labels. I suppose that a user who's only interested in primary lands will be able to reference this list to understand that they only need to set maxpatch for index 1.